buku

```diff
@@ -618,26 +621,37 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if not delete:
+                prompt(results, self.noninteractive, False)
```

Comment:
should be:

    prompt(results, self.noninteractive, delete)
we don't need to check the value of delete here with the new signature of `prompt`.